internal/services/containers/kubernetes_cluster_resource_test.go

```diff
@@ -18,8 +18,8 @@ type KubernetesClusterResource struct {
 }
 
 var (
-	olderKubernetesVersion   = "1.18.19"
-	currentKubernetesVersion = "1.19.11"
+	olderKubernetesVersion   = "1.19.11"
+	currentKubernetesVersion = "1.19.13"
```

Comment:
These should be a point release apart, latest non-preview is `1.21.2`, the prior release should probably be `1.20.9`